### PR TITLE
Admin dashboard

### PR DESCRIPTION
**Pull request by Eagerlearn**
* Add Admin Dashboard index page
* Take note of view folder location and the spec test location

### CODE REVIEW

**Comment by B-gann21** [comment]
Placement of the files looks appropriate to me. I like how you used the user story to structure the tests! Makes it readable and specific. Good work!

File: spec/features/admin/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Dashboard' do
+  describe 'As an Admin, when I visit the admin dashboard' do
```
Comment (CLRM1) [inline spec/features/admin/index_spec.rb:4]: I like this structure.

**Comment by CLRM1** [comment]
Looking good!

**Comment by amsalmeron** [comment]
Looks good to me. Sending a merge!

**Pull request merged**